### ISSUE TITLE
Css file size why large?

### ISSUE THREAD

**Issue by offwork** [ngneat/tailwind#98]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[√] Bug report  
[√] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

After the project is built, the css output size appears to be 7.5Mb, so I don't think purge for Tailwind is working correctly. 

**here is my tailwind config:**
```
module.exports = {
  prefix: '',
  purge: {
    content: ['./src/**/*.{html,ts}'],
  },
  darkMode: 'class', // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {
    extend: {},
  },
  plugins: [
    require('@tailwindcss/aspect-ratio'),
    require('@tailwindcss/forms'),
    require('@tailwindcss/line-clamp'),
    require('@tailwindcss/typography'),
  ],
};
```
**And output:**
<img width="656" alt="Screen Shot 2021-04-21 at 15 45 53" src="https://user-images.githubusercontent.com/10039583/115556387-4da15400-a2b9-11eb-8923-ca597eccd684.png">

## Minimal reproduction of the problem with instructions

For bug reports please provide the _STEPS TO REPRODUCE_ and if possible a _MINIMAL DEMO_ of the problem

## Environment

<pre><code>
Angular version: 11.2.7

Browser:
- [√] Chrome (desktop) version 89
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [√] Firefox version 88
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: 15.2.1  
- Platform:  darwin x64 

Others:

</code></pre>

**Comment by vltansky** [comment]
What is the command you are running prod build? E.g `ng b --prod`?

**Comment by vltansky** [comment]
try `ng b --prod`

**Comment by offwork** [comment]
Yes I didn't use the `--prod` flag, you are right @vltansky ! The problem's been solved.

**Comment by webben-de** [comment]
Same here +1

**Comment by vltansky** [comment]
@webben-de did prod flag solve it for you ?

**Comment by webben-de** [comment]
no this not! But i just figured out that prepend `NODE_ENV=production` to my build command solves it....
so for my case `"build": "NODE_ENV=production ng build --prod --stats-json",`

**Comment by KhoiNguyenICT** [comment]
Same here +1
